code/modules/arcade/arcade_base.dm

```diff
@@ -76,12 +72,22 @@
 			var/obj/item/stack/spacecash/C = O
 			if(pay_with_cash(C, user))
 				tokens += 1
-		return
-	if(panel_open && component_parts && istype(O, /obj/item/crowbar))
-		default_deconstruction_crowbar(user, O)
-		return
+			return
 	return ..()
 
+/obj/machinery/arcade/screwdriver_act(mob/living/user, obj/item/I)
+	if(!anchored)
+		return FALSE
+	default_deconstruction_screwdriver(user, icon_state, icon_state, I)
+	update_icon()
+	return TRUE
+
+/obj/machinery/arcade/crowbar_act(mob/living/user, obj/item/I)
+	if(!component_parts)
+		return FALSE
```

Comment:
Just as an optional suggestion thing, since it's so inconsistent in other machines that I don't know what the standard is, and it might end up "feeling" better without it:
```suggestion
	if(!component_parts || !panel_open)
		return FALSE
```
That'll allow people to still whack the claw machine with a crowbar when the panel is closed, rather than there just being no interaction.

Reply:
Yeah, will that check in.